tests/security/test_path_safety.py

```
@@ -0,0 +1,82 @@
+from pathlib import Path
```

Flake8 (7.3.0)

[error] 'pathlib.Path' imported but unused

(F401)